.github/workflows/pull-request.yml

```
@@ -0,0 +1,51 @@
+name: PR Build
+
+on: pull_request
+
+jobs:
+  ubuntu-build:
+    name: Build on Ubuntu
+    runs-on: ubuntu-latest
+    steps:
+      -   name: Checkout Repository
+          uses: actions/checkout@v1
```

Shall we use the latest version?

```suggestion
          uses: actions/checkout@v4
```

---

Fixed via [28dfba6](https://github.com/xlibb/module-pipeline/pull/1/commits/28dfba68bac4bf0a6d1e0166d49c7b45cc430d39)